.ci-mgmt.yaml

```diff
@@ -28,6 +28,7 @@ team: ecosystem
 goBuildParallelism: 2
 javaGenVersion: "v0.8.0"
 runner:
+  default: pulumi-ubuntu-8core
   prerequisites: pulumi-ubuntu-8core
   buildSdk: pulumi-ubuntu-8core
   publish: pulumi-ubuntu-8core
```

Comment:
Nit: We can comment these out if we want, since they are overriding the default *and* the same as the default.